### PR TITLE
Consistency on Success/Deliverables

### LINKED ISSUE #9: Where do we propose to advance to, CR or PR?
The charter seems to be confused as to whether we are going to try to go to PR or whether we are going to stop at PR.

### PR DESCRIPTION
**Pull request by seanturner**
Attempts to resolve #9.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -252,14 +252,22 @@ <h3>Timeline</h3>
 
 	<section id="success-criteria">
 	  <h2>Success Criteria</h2>
-	  <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
-
-	  <p>
-          Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+	  <p>The WG will progress its normative specificaitons through the following
```
Comment (martinthomson) [inline charter.html:255]:
```suggestion
	  <p>The WG will progress its normative specifications through the following
```

File: charter.html [hunk 2]
```diff
@@ -252,14 +252,22 @@ <h3>Timeline</h3>
...
+    standardization process: 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">First Public
+      Woring Draft</a>, 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working
+      Fraft</a>, and
```
Comment (martinthomson) [inline charter.html:260]:
```suggestion
      Draft</a>, and
```

**Review by martinthomson — APPROVED** [review]

File: charter.html [hunk 3]
```diff
@@ -252,14 +252,22 @@ <h3>Timeline</h3>
...
 	<section id="success-criteria">
 	  <h2>Success Criteria</h2>
-	  <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
-
-	  <p>
-          Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+	  <p>The WG will progress its normative specifications through the following
+    standardization process: 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">First Public
+      Woring Draft</a>,
```
Comment (martinthomson) [inline charter.html:258]:
```suggestion
      Working Draft</a>, 
```

File: charter.html [hunk 4]
```diff
@@ -252,14 +252,22 @@ <h3>Timeline</h3>
...
+	  <p>The WG will progress its normative specifications through the following
+    standardization process: 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">First Public
+      Working Draft</a>, 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working
+      Draft</a>, and
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsCR">Candidate
+      Recommendation</a>. 
+    It is expected that to reach the Candidate Recommendation stage, each normative
+    specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
```
Comment (jyasskin) [inline charter.html:264]: Here's the rough change I was suggesting in today's meeting, to ensure there's a clear phase (CR Draft) where the WG says "this is ready for implementation" that doesn't itself require the implementations to have already happened.

```suggestion
    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working
      Draft</a>,
    <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-draft">Candidate
      Recommendation Draft</a>, and
    <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot">Candidate
      Recommendation Snapshot</a>.
    It is expected that to reach the Candidate Recommendation Snapshot stage, each normative
    specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
```

@wseltzer @martinthomson @cwilso, what do you think?

It's probably worth raising this overall question with the Process CG, but that needn't block progress on this charter.

Comment (seanturner) [reply charter.html:264]: The one question I have is whether this needs to address the * on the [slide](https://www.w3.org/2020/Talks/TPAC/W3C2020/?full#7) to say there may be "0 or more".

Comment (martinthomson) [reply charter.html:264]: This seems fine to me.  I don't think that we need to worry about the 0 or more piece here.  I expect we'll use all of those states.  I can't see us creating a snapshot and then walking away with no further drafts.

Comment (seanturner) [reply charter.html:264]: I am fine with that just wanted to get an explicit "nope not needed" from somebody.

Comment (wseltzer) [reply charter.html:264]: Works for me

**Comment by AramZS** [comment]
👍 This looks good to me.

File: charter.html (hunk 4 above)
Comment (cwilso) [reply charter.html:264]: LGTM.

**Comment by seanturner** [comment]
I think this thread is at the point where it should basically be what we wanted from the last telecon. Please give feedback this week if you would like further changes. Otherwise I think we can consider this having achieved consensus. CC: @jyasskin  @AramZS  @martinthomson @ekr

**Pull request merged**